packages/manager/modules/pci/src/projects/project/training/dashboard/dashboard.component.js

```diff
@@ -0,0 +1,36 @@
+import controller from './dashboard.controller';
+import template from './dashboard.html';
+import './dashboard.scss';
+
+export default {
+  controller,
+  template,
+  bindings: {
+    usage: '<',
+    currencySymbol: '<',
+    trainingFeatures: '<',
+    registry: '<',
+    goToJobSubmit: '<',
+    goToDataCreate: '<',
+    goToDashboard: '<',
+    dashboardLink: '<',
+    goToRegistryAttach: '<',
+    registryAttachLink: '<',
+    goToRegistryDetach: '<',
+    registryDetachLink: '<',
+    installLink: '<',
+    jobList: '<',
+    refreshState: '<',
+    submitJobLink: '<',
+    saveRegistry: '<',
+    deleteRegistry: '<',
+    billingLink: '<',
+    jobInfo: '<',
+    jobKill: '<',
+    jobInfoLink: '<',
+    allUsers: '<',
+    userLink: '<',
+    goToJobs: '<',
+    regions: '<',
```

Comment:
Are all those needed at the root state ? It looks like a lot, not an issue it is just a wonder

Reply:
yes, the dasboard containss a lot of informations :)